packages/pirateship/src/components/PSHalfModal.tsx

```diff
@@ -13,7 +13,7 @@ export interface PSHalfModalProps {
   visible: boolean;
   renderLeftItem?: () => JSX.Element;
   renderRightItem?: () => JSX.Element;
-  children?: any;
+  children?: JSX.Element;
```

Comment:
You should be able to remove this property altogether -- `children` should come out-of-the-box with the `FunctionComponent` interface.